include/swift/Reflection/ReflectionContext.h

```diff
@@ -452,6 +452,9 @@ class ReflectionContext
     case MetadataSourceKind::SelfWitnessTable:
       return true;
     }
+
+    std::cerr << "fatal error: " << "Unhandled MetadataSourceKind in switch." << "\n";
+    std::abort();
```

Comment:
We get linker errors compiling swiftCore.dll if we use `llvm_unreachable`. The errors are `unresolved external llvm_unreachable_internal`. This is because we can't/don't link any LLVM libraries from the stdlib.